File: 001/sum.jl
function n_sum(n)
    total = 0
    for i in 1:n
        total = total + i
    end
    return total
end

start = time_ns()
for i in 1:10
    n_sum(1000000)
end
stop = time_ns()
print((stop-start)/10000000000)

File: 001/sum.py
import time
print("sum")
def n_sum(n):
    total = 0
    for i in range(1, n+1):
        total += i
    return total

start = time.time()
for _ in range(10):
    n_sum(1000000)
end = time.time()
print("python: ",(end-start)/10)